src/App.css

```diff
@@ -0,0 +1,112 @@
+body{
+  padding: 100px 250px 100px 250px;
+  background: #e5e5e5;
+  text-align: center;
+  font-family: 'Slabo 27px', serif;
+}
+.item{
+background: #f6f6f6;
+display: flex;
+flex-direction: row;
+}
+
+.item > div {
+  display: block;
+  padding: 5px 0 20px 0;
+  text-align: center;
+  width: 50%;
+  min-height: 60vh;
+}
+.itemlist{
+  border: 50px solid #3359db;
+}
+.header{
+  font-variant: unset;
+  color: #3359db;
+  text-transform: uppercase;
+}
+/* ul li {
+  padding: 20px;
+  text-align: left;
+  border-top: 1px solid black;
+  list-style: none;
+  
+}
+li{
+  padding-left: 16px;
+}
+li::before {
+  content: "-";
+  padding-right: 8px;
+  color: #455aa0;
+  font: 100px;
+  width: 400px;
+  
+} */
+table{
+  width: 100%;
+  color: #455aa0;
+  border-spacing: 0px !important;
+  font-size: 20px;
+}
+tr td {
+  padding: 15px;
+}
+td {
+  border: none;
+  border-top: 1px solid #3359db;
+  /* border-left: 1px solod #3359db; */
+}
+td {
+  text-align: left;
+  padding: 10px;
+  border-left: 1px solid #3359db;
+}
+
+hr.list{
+  border: 2px solid #3359db;
+  width: 30px;
+  padding: auto;
+  margin: auto;
+}
+
+hr{
+  border: 1px solid #3359db;
+  margin: auto;
+}
+
+/* to style the todo form component */
+.todo-form{
+  /* margin: 100px; */
+  display: flex inline;
+}
+.circle {
+  border-radius: 50%;
+  height: 130px;
+  width: 130px;
+  text-align: center;
+  margin:auto;
+  margin-top:60px;
```

Comment:
### Nitpick

Space your attribute and value. This should be `margin-top: 60px`. Same goes for `margin: auto` on top of this.